src/main/java/wolox/training/repositories/BookRepository.java

```diff
@@ -0,0 +1,10 @@
+package wolox.training.repositories;
+
+import org.springframework.data.jpa.repository.JpaRepository;
+import wolox.training.models.Book;
+
+public interface BookRepository extends JpaRepository<Book, Long> {
```

Comment:
The JpaRepository class adds functionality that is not needed. It would be better just to use Repository<Book, Long> instead.